libs/feed/clalit_feed.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC
+import json
+import requests
+from libs.feed.abstract import FeedAbstract
+
+from libs.feed.normalize.normalize_utils import normalize_telephones, normalize_treatments
+from libs.interfaces.document import Document, SourceType
+
+
+class ClalitFeed(FeedAbstract, ABC):
+
+    def pull(self) -> list[Document]:
+
+        documents: list[Document] = []
+        
+        url = 'https://apps.clalit.co.il/SupplierDataRESTService/api/supplier/GetJsonGroupSearchBody'
+
+        body_data = {
+            "serviceID": "33-0",
```

Comment:
Great job , there are more service id's that we can used ?

Reply:
To check this it needed more research.
For this url : https://mushlam.clalit.co.il/he/development/Pages/development_3-10_years.aspx?tab=5
there is 1209 records, all this records will get in to our db.